R/install.R

```diff
@@ -25,6 +25,9 @@ install_cmdstan <- function(dir = NULL, cores = 2, quiet = FALSE) {
     make_cmdstan <- c(make_cmdstan, paste0("-d ", dir))
   }
   make_cmdstan <- c(make_cmdstan, paste0("-j", cores))
+  if ((.cmdstanr$VERSION >= "2.21") && os_is_windows()) {
```

Comment:
I think (for now at least) we can just assume this function always installs the latest cmdstan release so just checking for windows is fine. Anyway, checking the version number would only tell us the version number they _already_ have installed (if any), right?

Reply:
Oh right, that is correct.